### PR TITLE
feat(dh): Convert playwright to cypress

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## Description

Convert playwright tests to Cypress E2E (with the testing library) and remove playwright-related code, dependencies, and tooling. 
This change fixes our broken e2e tests and, as a bonus, makes our e2e about 50% faster (~8min -> ~4min). 

## References

- #1654

### CODE REVIEW

File: apps/dh/e2e-dh/src/support/e2e.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
```
Comment (mimse) [inline apps/dh/e2e-dh/src/support/e2e.ts:32]: nitpick: could be removed

**Review by mimse — APPROVED** [review]
Great work!

File: apps/dh/e2e-dh/src/support/e2e.ts (hunk 1 above)
Comment (frontend-specialisten) [reply apps/dh/e2e-dh/src/support/e2e.ts]: Should be removed now :)

**Review by mimse — APPROVED** [review]

File: .github/workflows/app-dh-healthchecks.yml [hunk 2]
```diff
@@ -38,8 +38,5 @@ jobs:
       - name: 📥 Monorepo install
         uses: ./.github/actions/yarn-nm-install
 
-      - name: Install Playwright
-        run: npx playwright install --with-deps
-
       - name: Run healthchecks
-        run: npx nx run e2e-dh:health
+        run: npx nx run dh-cypress-e2e:e2e:health
```
Comment (dzhavat) [inline .github/workflows/app-dh-healthchecks.yml:42]: issue: looks like `dh-cypress-e2e` does not exist. Should probably be `e2e-dh:e2e:health`

**Pull request merged**